fall18/p3/project.py

```diff
@@ -0,0 +1,96 @@
+import csv
+
+class Project():

```

Comment:
Maybe we can do this without a class to make it simpler if students want to poke around.